lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -71,7 +76,7 @@ def preds_for_term
 
         def required_term_preds
           label_pred_uri = term_config.term_results_label_predicate
-          raise Qa::InvalidConfiguration, "required label_predicate is missing in configuration for LOD authority #{auth_name}" if label_pred_uri.nil?
+          raise Qa::InvalidConfiguration, "required label_predicate is missing in configuration for LOD authority #{authority_name}" if label_pred_uri.nil?
```

Comment:
This was a bug since `auth_name` was undefined.  Now uses delegation to full_config to get the authority name.